### PR TITLE
[ADD] website_header_language_switcher from superman

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: website_header_language_switcher/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (noahzaozao) [inline website_header_language_switcher/__openerp__.py]: header

**Comment by xie8899** [comment]
@noahzaozao @elicoidal @seb-elico Done

**Comment by noahzaozao** [comment]
@seb-elico Please review this PR , thanks

**Comment by seb-elico** [comment]
LGTM

**Pull request merged**